### ISSUE TITLE
--interleaved option doesn't work on v2.3.5

### ISSUE THREAD

**Issue by Changrim** [BenLangmead/bowtie2#240]
Hello.
--interleaved option doesn't work on v2.3.5.

I believe that this problem comes from bt2search.cpp / line 1651 to 1654

```C++
if(interleaved && (format != FASTA || format != FASTQ)) {
    cerr << "Error: --interleaved only works in combination with FASTA (-f) and FASTQ (-q) formats." << endl;
    throw 1;
}
```

So whatever value `format` has, this statement is `true` with --interleaved option, and it throws an error.

**Comment by ch4rr0** [comment]
Apologies. I pushed a fix for this issue. I will update the thread when the new builds are out. Thank you for reporting this.

**Comment by ch4rr0** [comment]
This change is available in v2.4.0 and later.
